azerothcore/azerothcore-wotlk
# Fix(Scripts/TrialOfTheCrusader):  solved issue #19834

fix(Scripts/TrialOfTheCrusader): Touch of light/Touch of darkness should not deal damage to players with same color


## Changes Proposed:

This PR proposes changes to:

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/19834

## SOURCE:

The changes have been validated through:

## Tests Performed:

This PR has been:


## How to Test the Changes:



1.Go to a fight with Twin Val'kyr
2.first player tanks 2 of them with dark/light debuff
3.wait second player to get debuff
4.second player should not get debuff "touch of light" with Light Essence or "touch of dark" with Dark Essence.

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
